content/en/security/cloud_security_management/setup/_index.md

```diff
@@ -109,4 +155,4 @@ To get started setting up CSM, navigate to the [**Security** > **Setup**][4] sec
 [3]: /security/cloud_security_management/setup/csm_workload_security
 [4]: https://app.datadoghq.com/security/configuration/csm/setup
 [5]: /security/identity_risks/#setup
-[6]: /security/infrastructure_vulnerabilities/setup
\ No newline at end of file
+[6]: /security/cloud_security_management/setup
```

Comment:
```suggestion
```

Reply:
Just noting that the link is pointing to this file, so it's essentially a redirect loop. But I also suggested we delete the line that included this link (above), so we should be fine deleting this as well.

Reply:
ok yeah.. there is no "dedicated CSM vulnerability setup" page now.. I see now on the other feedback above that this whole line can and should be removed. Noted.